src/overlays/actors/ovl_Bg_Jya_Zurerukabe/z_bg_jya_zurerukabe.c

```diff
@@ -83,14 +81,12 @@ void func_8089B4C8(BgJyaZurerukabe* this, GlobalContext* globalCtx) {
 
     if ((player->stateFlags1 == 0x200000) && (player->actor.wallPoly != NULL)) {
         s32 i;
-
         for (i = 0; i < ARRAY_COUNT(D_8089BA18); i++) {
             f32 posY = player->actor.world.pos.y;
             if ((posY >= D_8089BA18[i][0]) && (posY <= D_8089BA18[i][1])) {
                 break;
             }
         }
-
```

Comment:
I think most the spaces you removed actually improve readability, so it's unfortunate to remove them imo

Reply:
agreed, should add the, back

Reply:
and this as well actually